packages/teams-js/src/public/clipboard.ts

```diff
@@ -54,7 +55,7 @@ export namespace clipboard {
     if (!isSupported()) {
       throw errorNotSupportedOnPlatform;
     }
-    if (isHostClientMobile()) {
+    if (isHostClientMobile() || GlobalVars.hostClientType === HostClientType.macos) {
```

Comment:
is macOS being handled in a future PR? (sorry if you've already answered this)

Reply:
This change is just to add a check for macOS to the platform. I thought that macOS will be `framed` but actually it uses same as `iOS` which is `frameless` post messages.